README.md

```diff
@@ -58,12 +58,36 @@ type Copybook struct{
 
 where the values represent:
 ```go
-Dummy1 int      `pic:"4,1,4"`
+Dummy1 int      `pic:"4"` // start:1, end:4
```

Comment:
unsure what this does... can we chat about this before it goes in?

Reply:
👨🏼‍🍳 💋